### PR TITLE
Completed calculator with tests

### PR DESCRIPTION
**Pull request by malbrecht0792**
Created a basic calculator app with tests

### CODE REVIEW

File: Homework_01/Calculator/spec/calculator_spec.rb [hunk 1]
```diff
@@ -0,0 +1,49 @@
+require_relative '../calculator'
+
+describe 'Calculator' do
+
+    before(:each) do
```
Comment (creid31) [inline Homework_01/Calculator/spec/calculator_spec.rb:5]: Great job using before each here. How would this be different with FactoryBot?

File: Homework_01/Calculator/spec/calculator_spec.rb [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    it 'multiplies two numbers' do
+        expect(@calculator.multiply(4,3)).to eq(12)
+    end
+
+    it 'has a divide method' do
+        expect(@calculator).to respond_to(:divide) 
+    end
+
+    it 'multiplies two numbers' do
+        expect(@calculator.divide(6,3)).to eq(2)
```
Comment (creid31) [inline Homework_01/Calculator/spec/calculator_spec.rb:46]: How would your code or tests change to prevent a user from trying to divide by 0 which could cause errors in your program?

**Review by creid31 — APPROVED** [review]

File: Homework_01/Calculator/spec/calculator_spec.rb (hunk 1 above)
Comment (malbrecht0792) [reply Homework_01/Calculator/spec/calculator_spec.rb:5]: Thanks! With FactoryBot, you would declare a calculator factory and then in the spec use a let statement to assign calculator to build(:calculator) so that you can use the calculator variable in the tests

File: Homework_01/Calculator/spec/calculator_spec.rb (hunk 2 above)
Comment (malbrecht0792) [reply Homework_01/Calculator/spec/calculator_spec.rb:46]: I would write another test that displays an error if the user divides by zero. I would expect that when the calculator method is given a divisor of zero, it will throw an error saying you can't divide by zero.
